GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -39,14 +39,57 @@ namespace gvr {
 
 static int numberDrawCalls;
 static int numberTriangles;
+static float drawTime;
+static float avgDrawTime;
+static bool monoscopic = false;
+static bool disjointTimerSupported = false;
+static int currentFrameQuery;
+static int lastFrameQuery;
+static GLuint queries[2][2];
+static GLuint queriesAvailable;
+static GLint queriesDisjointOccurred;
+static GLuint64 timeElapsed;
+static GLuint64 frameCount;
+
+#ifndef GL_TIME_ELAPSED_EXT
+#define GL_TIME_ELAPSED_EXT 0x88BF
+#define GL_GPU_DISJOINT_EXT 0x8FBB
+extern "C" {
+typedef void (*PFNGLGETQUERYOBJECTUI64VEXTPROC) (GLuint id, GLenum pname, GLuint64 *params);
+PFNGLGETQUERYOBJECTUI64VEXTPROC glGetQueryObjectui64vEXT__;
+}
+#endif
 
 void Renderer::initializeStats(){
-    // TODO: this function will be filled in once we add draw time stats
+    currentFrameQuery = 0;
+    lastFrameQuery = 0;
+    timeElapsed = 0;
+    frameCount = 0;
+    drawTime = 0.0f;
+    avgDrawTime = 0.0f;
+
+    const GLubyte *extensions = glGetString(GL_EXTENSIONS);
+    char *exist = strstr((char *)extensions, "GL_EXT_disjoint_timer_query");
+    if(!strncmp(exist, "GL_EXT_disjoint_timer_query", 26)) {
```

Comment:
I'm having trouble figuring out what this code means to do. First, it calls `strstr()` to get a pointer to the start of a substring. Then it passes that pointer - which may be `NULL` - to `strncmp()` with a `size_t n` argument of 26.

Why 26? I mean 1) I count 27 chars in `"GL_EXT_disjoint_timer_query"` and 2) why a magic number?

What is the point of the `strncmp()` call?